f2/src/utils/formatAnalystEmail.js

```diff
@@ -40,7 +40,8 @@ const formatReportInfo = (data) => {
   delete data.reportId
   delete data.submissionTime
   delete data.language
-  delete data.appVersion
+  delete data.appVersion // git hash not used in report
+  delete data.prodVersion
```

Comment:
prodVersion is still showing in missing fields. I wonder if initialState is interfering with this...